notebooks/multimodal_vdj/01_preprocessing.ipynb

```diff
@@ -0,0 +1,3169 @@
+{
```

Comment:
<p>Might need to explain "contigs" here? I can follow, but newbies can't.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>